src/scheduler.rs

```diff
@@ -663,6 +632,10 @@ async fn fetch_idle(
         .log_err(ctx)
         .ok();
 
+    download_msgs(ctx, &mut session)
```

Comment:
I think it's better to move this right after `download_known_post_messages_without_pre_message()` so that it goes before `scan_folders()`. Scanning folders isn't that important, it's even debounced to once per minute